Rivet/Rivet.psm1

```diff
@@ -177,6 +221,7 @@ Test-RivetTypeDataMember -TypeName 'Rivet.Scale' -MemberName 'Value'
     Join-Path -Path $rivetModuleRoot -ChildPath 'Functions'
     Join-Path -Path $rivetModuleRoot -ChildPath 'Functions\Columns'
     Join-Path -Path $rivetModuleRoot -ChildPath 'Functions\Operations'
+    Join-Path -Path $rivetModuleRoot -ChildPath 'Functions\Metadata'

```

Comment:
What's this `Functions\Metadata` directory?

Reply:
I was going to start introducing metadata functions into Rivet (e.g. Get-ExtendedProperty, Get-Table, etc.) for use in plug-ins, but decided they fit better in Yodel than in Rivet, so removed them. Looks like I missed this spot.